R/epidatacall.R

```diff
@@ -86,9 +87,9 @@ request_arguments <-
     for (name in names(all_params)) {
       v <- all_params[[name]]
       if (!is.null(v)) {
-        if (inherits(v, "EpiRange")) {
+        if (test_class(v, "EpiRange")) {
```

Comment:
note: probably doesn't matter here, because we shouldn't be running this in a tight loop, but `test_*` functions look very slow, e.g., 4x--17x the time.  Due to (a) being implemented on top of check functions that also might prepare an error message (which `test_*` converts to `FALSE`), and (b) that have more features (that we don't use, but which probably have significant cost relative to the basic check even when not used).

Reply:
Right, I am choosing consistency here